Dockerfile

```diff
@@ -1,16 +1,15 @@
-ARG DOTNET_ARCH=
-
-FROM node:lts AS build-node
+FROM --platform=$BUILDPLATFORM node:lts AS build-node
```

Comment:
I think we can use `lts-alpine` instead of `lts` - it is 10 times lighter and will make build a little bit faster, not significantlly though. However it requires a dependency of git (since webpack needs to get commit hash) to be installed.

Reply:
See https://github.com/JustArchiNET/ArchiSteamFarm/pull/1238 for my rationale why I dislike `alpine` as it uses `musl` instead of `glibc`.

Reply:
In this context: since we use `node` for compilation and `musl` performs worse on `x64`, there is a premise of unnecessarily increasing compile time for no real advantage, as the size increase is negligible when confronted with caching and GitHub's gigabits links.